### ISSUE TITLE
H4 color passes color contrast only for white background—fails for light gray and light blue

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/alasdoo_com_v2#132]
Change the styling when all the heading styles are updated
